integration/tests/cook/test_basic.py

```diff
@@ -1640,14 +1640,16 @@ def num_running_predicate(response):
                 num_jobs_total = len(response)
                 num_running = len([j for j in response if j['status'] == 'running'])
                 num_waiting = len([j for j in response if j['status'] == 'waiting'])
+                self.logger.info(f'There are {num_jobs_total} total jobs, {num_running} running jobs, '
+                                 f'and {num_waiting} waiting job(s)')
                 if num_jobs_total == num_hosts + 1:
                     # One job should not be scheduled
                     return (num_running == num_jobs_total - 1) and (num_waiting == 1)
                 else:
                     # All of the jobs should be running
                     return num_running == num_jobs_total
 
-            jobs = util.wait_until(query, num_running_predicate, max_wait_ms=60000)
+            jobs = util.wait_until(query, num_running_predicate)
```

Comment:
Why did the timeout change?

Reply:
60 seconds isn't enough in some of our environments, and if you use the default, you can change it with an environment variable, which is nice

Reply:
👍